File: go-sdk/pkg/models/shared/sharedorganization.go
package shared

// SharedOrganization
// The organization the resource belongs to
type SharedOrganization struct {
	Name           *string `json:"name,omitempty"`
	ProfileImage   *string `json:"profile_image,omitempty"`
	ProfileImage90 *string `json:"profile_image_90,omitempty"`
	Slug           *string `json:"slug,omitempty"`
	Username       *string `json:"username,omitempty"`
}

File: ts-sdk/src/sdk/models/shared/sharedorganization.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// SharedOrganization
/** 
 * The organization the resource belongs to
**/
export class SharedOrganization extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=name" })
  name?: string;

  @SpeakeasyMetadata({ data: "json, name=profile_image" })
  profileImage?: string;

  @SpeakeasyMetadata({ data: "json, name=profile_image_90" })
  profileImage90?: string;

  @SpeakeasyMetadata({ data: "json, name=slug" })
  slug?: string;

  @SpeakeasyMetadata({ data: "json, name=username" })
  username?: string;
}
